requirements-frozen.txt

```diff
@@ -1,4 +1,6 @@
 arabic-reshaper==2.1.1
+astroid==2.5.6
```

Comment:
perhaps it would be better to create a requirements-dev file which includes dependencies for development, and not for deploy.

Reply:
I split the two apart, which also removed the merge conflict. That should suffice for now. In the long run, poetry may be an option, but one step at a time. :)